libs/util/src/lib/extractIds.ts

```diff
@@ -0,0 +1,44 @@
+export enum extractIdType {
+  tournament = 'tournament',
+  player = 'player',
+  race = 'race',
+  playerLog = 'playerLog',
+}
+
+export const extractId = (
```

Comment:
Please do the type casts to the returned type, so that the consumers of this function will not have to do manual type casts.

Reply:
Solved in PR #14 under commit 42e5c196dd4d85bfe7c232dd718c2b5c5935231e